MetaMask/metamask-design-system
# feat: initialize storybook react ui

## **Description**

In this pull request, we introduced the initial setup for Storybook documentation for React components. The documentation was added inside the `docs` folder. This was done using the `create package` command, which automatically generated some files. These files were later moved into the `docs` folder. Additionally, placeholder comments for TODOs were left in the files to be addressed when the components package is created. Some follow-up changes to the Storybook setup will also be required in the future.

## **Related issues**

#729

## **Manual testing steps**

1. Navigate to the Storybook setup in the `docs` folder.
2. Verify that the generated documentation reflects the correct structure and includes the appropriate placeholders.
3. Ensure that any follow-up TODOs are clearly marked for future reference.

## **Screenshots/Recordings**

### **Before**
n/a


### **After**

https://github.com/user-attachments/assets/a2d42876-a32f-4ff0-8301-713430a831bf




## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
